api/src/paths/dwc/submission/queue.ts

```diff
@@ -0,0 +1,118 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SOURCE_SYSTEM } from '../../../constants/database';
+import { getServiceAccountDBConnection } from '../../../database/db';
+import { HTTP400 } from '../../../errors/http-error';
+import { defaultErrorResponses } from '../../../openapi/schemas/http-responses';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { SubmissionJobQueueService } from '../../../services/submission-job-queue-service';
+import { scanFileForVirus } from '../../../utils/file-utils';
+import { getKeycloakSource } from '../../../utils/keycloak-utils';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/dwc/submission/queue');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          validServiceClientIDs: [SOURCE_SYSTEM['SIMS-SVC-4464']],
+          discriminator: 'ServiceClient'
+        }
+      ]
+    };
+  }),
+  queueForProcess()
+];
+
+POST.apiDoc = {
+  description: 'Submit DwCA to be queued and processed',
+  tags: ['dwca'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  requestBody: {
+    content: {
+      'multipart/form-data': {
+        schema: {
+          type: 'object',
+          required: ['media', 'dataset_uuid'],
```

Comment:
thoughts on changing `dataset_uuid` to `data_package_id`, to be consistent with other endpoints?

Reply:
renamed to `data_package_id` to keep things consistent